src/spotify.service.ts

```diff
@@ -1,19 +1,21 @@
 import SpotifyWebApi from 'spotify-web-api-node';
 import { waitForCode } from './auth-server';
-import config from './config.json';
+import 'dotenv/config'
+import env from 'env-smart';
 import SpotifyAuth from './spotify-auth';
 import fs from 'fs';
 import { response } from 'express';
+env.load();
 
```

Comment:
Would it be possible to have all the necessary config declared in the imports like this
`const { SPOTIFY_CLIENT_ID, SPOTIFY_CLIENT_SECRET, etc } = process.env`
Personally I think it makes it look a bit nicer just so there isn't a million `process.env.XYZ` spread throughout the code